changelogs/fragments/1780-zos_mvs_raw-verbose-fail.yml

```
@@ -0,0 +1,8 @@
+bugfixes:
+  - zos_mvs_raw - If a program failed with a non-zero return code and verbose was false, the module would succeed.
```

```suggestion
  - zos_mvs_raw - If a program failed with a non-zero return code and verbose was false, the module would succeed (false positive).
```

i'm not sure if this suggested adds any clarity (keep it if it does 😄)

---

it does! thanks